### PR TITLE
Export event args types from Types.tsx as well so they can be mentioned in apps

### PR DESCRIPTION
**Pull request by asklar**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-xaml/pull/186)

### CODE REVIEW

File: package/Codegen/TSTypes.cs [hunk 1]
```diff
@@ -18,8 +18,8 @@ namespace Codegen
     /// Class to produce the template output
     /// </summary>
     
-    #line 1 "F:\react-native-xaml\package\Codegen\TSTypes.tt"
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
+    #line 1 "C:\Users\asklar\source\repos\react-native-xaml\package\Codegen\TSTypes.tt"
```
Comment (jonthysell) [inline package/Codegen/TSTypes.cs:21]: Do you want your name coded in here?

File: examplenuget/App.tsx [hunk 2]
```diff
@@ -101,57 +101,10 @@ const App = () => {
...
           />
 
-          <Button
-            onTapped={a => {setShowState(ContentDialogState.Popup);}}
-            content="click to open a ContentDialog" />
+        <WinUI.ColorPicker onColorChanged={(e) => {
+          console.log(e.nativeEvent);
+        }} />
 
-          <ContentDialog
```
Comment (jonthysell) [inline examplenuget/App.tsx:108]: Does this part of the example no longer work? Or just no longer matter?

**Review by jonthysell — APPROVED** [review]

File: package/Codegen/TSTypes.cs (hunk 1 above)
Comment (asklar) [reply package/Codegen/TSTypes.cs:21]: yeah this is autogenerated code, the `#line` directive is just there for help debugging

File: examplenuget/App.tsx (hunk 2 above)
Comment (asklar) [reply examplenuget/App.tsx:108]: it's just a playground kind of app, so the code in here doesn't quite matter :)

**Pull request merged**